html/webappapis/scripting/processing-model-2/compile-error-data-url.html

```diff
@@ -17,11 +17,11 @@
         ran = true;
         col_value = d;
         assert_equals(typeof a, 'string', 'first arg');
-        assert_equals(b, 'data:text/javascript,for(;) {}', 'second arg');
+        assert_equals(b, 'data:text/javascript,for(;){}', 'second arg');
         assert_equals(typeof c, 'number', 'third arg');
     });
   </script>
-  <script src="data:text/javascript,for(;) {}"></script>
```

Comment:
(Actually I think the spec says to not percent-encode the space either, but that's not what this test is trying to test.)